foreman/Dockerfile

```diff
@@ -13,6 +13,12 @@ COPY data_models/dist/data-refinery-models-* data_models/
 RUN pip install \
     data_models/$(ls data_models -1 | sort --version-sort | tail -1)
 
+COPY common/dist/data-refinery-common-* common/
```

Comment:
I believe I did this the way I did because what you're describing isn't possible. The only Dockerfile command capable of interacting with the host's filesystem is `COPY`, so until I copy the directory into the image I cannot run `$(ls data_models -1 | sort --version-sort | tail -1)` on it.